framework/utils/xmlUtils.py

```diff
@@ -206,16 +206,16 @@ def fixXmlText(msg):
     @ Out, msg, string, fixed string
   """
   #if not a string, pass it back through
-  if not isinstance(msg,basestring):
+  if not type(msg).__name__ in ['unicode','str']:
```

Comment:
This is at least how I got around the issue when breaking the classes into their own submodules.  See, for instance in `PointSet.py`:

```python
from .Data import Data, NotConsistentData, ConstructError
```